netbox/dcim/forms/bulk_import.py

```diff
@@ -256,6 +256,13 @@ class RackImportForm(NetBoxModelImportForm):
         to_field_name='name',
         help_text=_('Name of assigned role')
     )
+    rack_type = CSVModelChoiceField(
```

Comment:
`width` and `u_height` should no longer be required if `rack_type` is supported (although the form should have logic to check that these are set if `rack_type` is not).

Reply:
I honestly don't know how to show in the view the "OR" option --> ( `width` and `u_height`  mandatory  and rack_type optional ) "OR"  (`rack_type` and ( `width` and `u_height`  optional ) )
Or you prefer to have `width` and `u_height` not required anymore in the form AND add validation/check on values passed?

Reply:
We just need to mark them as `required=False` and add some conditional validation logic. I've addressed this with the latest commit.